src/NServiceBus.Callbacks.Testing/NServiceBus.Callbacks.Testing.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net452;netstandard2.0;net6.0</TargetFrameworks>
```

@soujay This is a NuGet package that is published by this repo containing support for running callback-related tests. Nothing that includes `netstandard2.0` as a target should also have `net6.0`.
```suggestion
    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
```